libs/partners/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -138,6 +138,30 @@ class Milvus(VectorStore):
         metadata_field (str): Name of the metadta field. Defaults to None.
             When metadata_field is specified,
             the document's metadata will store as json.
+        metadata_schema  (Optional[dict]): What is the dataType of each metadata fields,
```

Comment:
What if you received a Optional[CollectionSchema] instead?

Reply:
Yes,That's a good idea. need to think about other parameter which we are adding in schema while embedding(Size of vector we can get only after getting embedding).

i guess idea of schema creation in langchain milvus to match metadata field with schema, and less chances of error while inserting data in milvus.